### ISSUE TITLE
Run bbc-a11y in CI

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#305]
**Is your feature request related to a problem? Please describe.**
We should run BBC a11y against our article page. 
https://github.com/bbc/bbc-a11y 

**Describe the solution you'd like**
We should use bbc-a11y cli tool and run it against article pages on every PR, via Travis CI https://github.com/bbc/bbc-a11y
We should agree on a threshold - i.e. no errors - and fail the build if it introduces accessibility regressions.

**Comment by jamesbhobbs** [comment]
We should extend to use https://github.com/bbc/a11y-tests-web but not using docker in CI. We should also extend https://github.com/bbc/a11y-tests-web to conditionally enable lighthouse tests andymsuk will field PRs
